### ISSUE TITLE
Blob properties dialog: no visible or audible error message when there is no metadata name or value

### ISSUE THREAD

**Issue by MRayermannMSFT** [microsoft/AzureStorageExplorer#239]
**Storage Explorer Version**: 1.2.0
**Platform**: Windows
**Architecture**: win32
**Regression From**: Unknown

#### Steps to Reproduce: ####
1. Open the properties dialog for a blob with 
2. Tab to the add metadata button
3. Hit enter
4. Tab up to the value text box and enter a value
5. Tab to the name text box and delete it

#### Expected Experience: ####
There should be a text error message that says the metadata needs a name. The same error should also be ready the screen reader.

#### Actual Experience: ####
There's neither of those! 😢

**Comment by v-xuanzh** [comment]
This issue has been fixed. So close it.
